src/Jira/Api.php

```diff
@@ -198,6 +198,17 @@ public function editIssue($issue_key, array $params)
 	}
 
 	/**
+	 * Gets attachments meta information.
+	 *
+	 * @return array
+	 * @since  2.0.0
+	 */
+	public function getAttachmentsMetaInformation()
+	{
+		return $this->api(self::REQUEST_GET, '/rest/api/2/attachment/meta');
+	}
+
+	/**
 	 * Gets attachment.
```

Comment:
Isn't getAttachmentsMetaInformation already merged #101? Please remove from this diff